docs/features/ValueTuples.cs

```diff
@@ -24,8 +24,89 @@ public interface ITuple
         object this[int i] { get; }
     }
 
-    public static class ValueTuple
+    public struct ValueTuple : IEquatable<ValueTuple>, IStructuralEquatable, IStructuralComparable, IComparable, ITupleInternal, ITuple
     {
+        public override Boolean Equals(Object obj)
```

Comment:
I looked at the rest of the corefx codebase and it does indeed use the simplified types. I'm not sure why the code on the reference site used `Object` and `Boolean`... 
Fixed.